### DISCUSSION TITLE
Ability to change check order

### DISCUSSION THREAD

**Discussion by jazzdan** [community/community#16217] [Pull Requests]
Right now I don't think there is a way to order the checks that appear in a pull request. It would be nice to be able to change the order so that I can surface tests that frequently fail first.

**Comment by flaksp** [comment]
Option to sort them alphabetically or group them by workflows will be a nice UX improvement for PRs for lots of checks I believe.

**Comment by jacekmaciag** [comment]
Ordering by status would be fantastic. We have a bunch of checks that get skipped and it'd be great if they were listed somewhere at the bottom.

**Comment by TosinAF** [comment]
+1 to this.

**Comment by minh-gfg** [comment]
+1 would be appreciated for this.
cc @ptphuc

**Comment by ViliusS** [comment]
We are running GitHub organization integrated with Jenkins CI through GitHub Checks API. This is an usual view which we see:
![image](https://github.com/community/community/assets/2323565/bd98c2d0-c0c8-4778-966c-b36cdc5670e1)

"Jenkins Pipeline" check is the main check which has queued, in progress and completed statuses.
"Java", "Maven" and other possible checks are just optional checks which complete at the end of the pipeline with completed status.

It would be great to have "Jenkins Pipeline" first in the list, and all other checks below. Of course, we could name first check to something like "_Jenkins Pipeline" so alphabetically it is first, but that would be a workaround, not the final solution. Hence, ordering of the checks in UI (via some kind of defined category or grouping) would be great.
